wasm.md

```diff
@@ -120,6 +120,59 @@ When a binary operator is the next instruction, the two arguments are loaded fro
          <stack> < ITYPE > C2 : < ITYPE > C1 : STACK => STACK </stack>
 ```
 
+### Test Operations
+
+Test operations consume one operand and produce a bool, which is an `i32` value.
+They are parametric in the operand type.
+For each element added to `ITestOp`, function `#iTestOp` must be defined over it.
+Since test operations can not trap, it suffices to lift the values into a function.
+
+```k
+    syntax Instr  ::= "(" IValType "." ITestOp ")"
+    syntax Int    ::=  #iTestOp(ITestOp, IValType, Int) [function]
+ // --------------------------------------------------------------
+    rule <k> ( ITYPE . TOP:ITestOp ) => . ... </k>
+         <stack> (< ITYPE > SI1 => < i32 > #iTestOp(TOP, ITYPE, SI1)) : STACK </stack>
+```
+
+### Comparison Operations
+
+Comparisons consume two operands and produce a bool, which is an `i32` value.
+They are parametric in the operand type.
+For each element added to `RelOp`, function `#relOp` must be defined over it.
+Since comparison operations can not trap, it suffices to lift the values into a function.
+
+
+```k
+    syntax Instr  ::= "(" IValType "." RelOp ")"
+    syntax RelOp  ::= IRelOp
+ //                 | FRelOp
+    syntax Number ::= #relOp(RelOp, IValType, Int, Int) [function]
```

Comment:
This will always return an `Int` right?

Reply:
Yep, my bad. It doesn't matter now though, `#relOp` is removed.